content/en/docs/Badges/MCH2022/software-development/rp2040/_index.md

```diff
@@ -5,33 +5,35 @@ nodateline: true
 weight: 1
 ---
 
+## Introduction
 
-# This voids your warranty!
-# SERIOUSLY WE WON'T SUPPORT YOU IF YOU BRICK YOUR BADGE
+RP2040: aka Raspberry Pico. This is an onboard coprocessor that handles
+two USB <-> serial bridges and acts as an IO extender.
 
-# Introduction
+At this point in time, we have no way for the apps to include custom
```

Comment:
This is a bit weird, because the very next section is called "Install custom firmware" which is it? No way to install custom firmware or ...

Reply:
One cannot bundle RP2040 firmware with apps, but one can install custom firmware the same way easily the way described.

Reply:
Perhaps this way:... no way for the apps to automatically load new firmware to the RP2040 along with their main functionality on the ESP32, but manually flashing a custom firmware using the recovery method is possible, although not recommended.

Reply:
Done in fresh commit.